src/pages/about/About.tsx

```diff
@@ -16,9 +16,9 @@ export const About = (): ReactElement => {
   return (
     <Dialog title="About" open onClose={onDialogClose}>
       <img
-        className="border-white mb-8 h-64 rounded-lg border-4"
+        className="border-white mb-8 rounded-lg border-4 min-w-img-md min-h-img-md aspect-ratio-16/9"
```

Comment:
The `h-64` was important here to fix the height. Now the image is bigger than this size. That being said, fixing the image height previously was not responsive so on small screens the image would stretch vertically, a bug. Let's improve this by removing the class names added in the PR and instead adding:

```
w-full max-w-64 min-h-32
```

My issue was missing some context but this image is square so `max-w-64` will limit our width while still remaining responsive. `min-h-32` will simply add some height while the image is loading so it does not appear squashed.